tests/Microsoft.Identity.Test.Unit/BrokerTests/BrokerRequestTests.cs

```diff
@@ -547,6 +550,7 @@ public void SpecialAccount_CallsBrokerSilentAuth()
                 mockBroker.IsBrokerInstalledAndInvokable().Returns(false);
 
                 var platformProxy = Substitute.For<IPlatformProxy>();
+                platformProxy.CanBrokerSupportSilentAuth().Returns(true);
```

Comment:
>platformProxy.CanBrokerSupportSilentAuth().Returns(true); [](http://example.com/codeflow?start=16&length=57)

seems like we have no utest validating the true state.... shouldn't that be possible to test in the e2e tests running on iOS?

Reply:
That will require iOS. So I would do it manually.